resources/repository.rb

```diff
@@ -46,6 +46,9 @@
     package 'apt-transport-https'
     package 'dirmngr' if (node['platform'] == 'debian' && node['platform_version'].split('.')[0].to_i >= 9) || (node['platform'] == 'ubuntu' && node['platform_version'].split('.')[0].to_i >= 18)
     apt_key = new_resource.apt_gpg_key == 'F1656F24C74CD1D8' && node['platform'] == 'debian' && node['platform_version'].split('.')[0].to_i < 9 ? 'CBCB082A1BB943DB' : new_resource.apt_gpg_key
+    if node['lsb']['codename'] == 'buster' || node['lsb']['codename'] == 'buster/sid'
```

Comment:
If someone tries to override this in the resource this change will overwrite their new_resource.apt_gpg_key
Can we change this to be part of the top line, or add logic into the defaults? e.g. something like: https://github.com/sous-chefs/haproxy/blob/master/resources/install.rb#L23

Reply:
Good catch. 👍